deployment/GI01SUMO_Deploy.csv

```diff
@@ -35,73 +35,73 @@ KN-221-4,,AT-30-01,,GI01SUMO-RII11-02-CTDMOR020,1,1,2014-09-10T18:50:00,2015-08-
+AT-30-01,,AR-07-01,,GI01SUMO-RII11-02-PHSENE041,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-PHSENE-P0138,59.94358,-39.57372,,20,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-RII11-02-PHSENE042,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-PHSENE-P0139,59.94358,-39.57372,,100,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBC11-00-CPMENG000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGCON-BCPM01-50001,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-00-DCLENG000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGCON-BDCL11-50001,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-01-MOPAK0000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-MOPAK0-99999,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-03-HYDGN0000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-HYDGN0-99999,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-04-DOSTAD000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-DOSTAD-00456,59.94358,-39.57372,,1,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-05-SPKIRB000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-SPKIRB-00288,59.94358,-39.57372,,-6,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-06-METBKA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-METLGR-00021,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD11-08-NUTNRB000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-NUTNRB-00274,59.94358,-39.57372,,1,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-00-DCLENG000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGCON-BDCL12-50001,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-01-OPTAAD000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-OPTAAD-00242,59.94358,-39.57372,,1,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-02-FLORTD000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-FLORTD-01296,59.94358,-39.57372,,1,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-03-HYDGN0000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-HYDGN0-99999,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-04-PCO2AA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-PCO2AA-34233,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-05-WAVSSA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-WAVSSA-05321,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-06-METBKA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-METLGR-00019,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-08-FDCHPA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-FDCHPA-44903,59.94358,-39.57372,,-6,2668,

```

Comment:
Deployment 2 CTDMOQ017 depth should be 500 (not 501).

Reply:
fixed

Reply:
Looks good.